.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Release
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      tag:
+        description: "Tag name, e.g. 0.0.0"
+        default: ""
+        required: true
+
+jobs:
+  build:
+    name: Upload Release Asset
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Git set user
+        run: |
+          git config  user.name "$USERNAME"
+          git config  user.email "$USERNAME-[bot]@users.noreply.github.com"
+        env:
+          USERNAME: ${{ github.actor }}
+
+      - name: Update the VERSION
+        run: |
+          echo "$VERSION" > VERSION
```

Comment:
I am not sure, if we can use shell script to do a math function. for example read 0.0.3 from current file auto increases to 0.0.4 and commit?

Reply:
hmm, we can do that as well. But I wanted the version to be provided during runtime in cases where we want to break the continuous flow and maybe release something like `0.1.0`